workshop_3/workshop_3.ino

```diff
@@ -0,0 +1,56 @@
+#include <Adafruit_Sensor.h> // include these three libraries (Adafruit_Sensor.h, DHT.h, DHT_U.h)
+#include <DHT.h> 
+#include <DHT_U.h>
+DHT_Unified dht(2, DHT11);   // to define that sensor DHT11 is on pin 2
+const int RedLedPin = 13; // to define that the red LED is on pin 13
+const int GreenLedPin = 11; // to define that the green LED is on pin 11
+const int MoisturePin = A0; // to define that moisture sensor is on pin A0
+const int YellowLedPin = 12; // to define that the yellow LED is on pin 12
+int Moisturevalue; //  to define the variable that represents the moisture's value
+int DryValue = 466; // to define that the dryness threshold is set at 466. All results/values over 466 are considered as dry 
```

Comment:
Variable name not ok => dryValue

Reply:
It is not clear to have the value here. Prefer to have a minValue, a maxValue and apply a percentage of 20% to understand how you find that value